scripts/check_all_tests_are_covered.py

```diff
@@ -37,7 +37,7 @@ def from_yaml():
     # First collect the matrix definitions from testing workflows
     matrices = {}
     for wf in ["tests.yml"]:
-        wfname = wf.strip(".yml")
+        wfname = wf.removesuffix(".yml")
```

Comment:
This was only introduced in `3.9` https://docs.python.org/3/library/stdtypes.html#str.removesuffix

Reply:
Good catch!